pyart/graph/radardisplay.py

```diff
@@ -708,7 +774,19 @@ def plot_colorbar(self, mappable=None, field=None, label=None, cax=None,
             if field is None:
                 field = self.plot_vars[-1]
             label = self._get_colorbar_label(field)
-        cb = fig.colorbar(mappable, ax=ax, cax=cax)
+
+        if orient is None:
+            orient = 'vertical'
+
+        # Find the axes locations to set colorbar
+        if cax is None:
```

Comment:
I do not like the idea of having fixed values for size and pad.  plt.colorbar will resize the parent axes if cax is None will a "good" size and padding.  I would rather use matplotlib defaults than make our own.  @nguy Is there a good reason for the other custom padding and sizing of colorbars?